AdventOfCode.Library/Day02/Part2.fs

```diff
@@ -0,0 +1,62 @@
+module AdventOfCode.Library.Day02.Part2
+
+let seqToTuple (xs: 'T seq) : 'T * 'T = (Seq.item 0 xs, Seq.item 1 xs)
```

Comment:
I would like to make this return `option` if the sequence is too short or too long.